src/main/webapp/app/detail-overview-list/detail-overview-list.component.html

```diff
@@ -0,0 +1,264 @@
+                                            <fa-icon [icon]="faExclamationTriangle" class="text-warning me-1" [ngbTooltip]="noCheckoutDirectorySetTooltip"></fa-icon>
+                                            <span jhiTranslate="artemisApp.programmingExercise.noCheckoutDirectorySet">Checkout Directory was not set</span>
+                                        }
+                                        <ng-template #noCheckoutDirectorySet>
+                                            <fa-icon [icon]="faExclamationTriangle" class="text-warning me-1" [ngbTooltip]="noCheckoutDirectorySetTooltip"></fa-icon>
+                                            <span jhiTranslate="artemisApp.programmingExercise.noCheckoutDirectorySet">Checkout Directory was not set</span>
+                                        </ng-template>
+                                        <ng-template #noCheckoutDirectorySetTooltip>
+                                            <span jhiTranslate="artemisApp.programmingExercise.noCheckoutDirectorySetTooltip"></span>
+                                        </ng-template>
+                                    </div>
+                                </li>
+                            }
+                        </ul>
+                    </dd>
+                }
+                @case (DetailType.ProgrammingTestStatus) {
+                    <dd>
+                        @if (detail.data.participation) {
+                            <div class="d-flex align-items-center">
+                                @if (!detail.data.loading) {
+                                    <jhi-updating-result
+                                        [exercise]="detail.data.exercise"
+                                        [participation]="detail.data.participation!"
+                                        [showUngradedResults]="true"
+                                        [personalParticipation]="false"
+                                        [short]="false"
+                                        (onParticipationChange)="detail.data.onParticipationChange()"
+                                        class="me-2"
+                                    ></jhi-updating-result>
+                                }
+                                @if (detail.data.participation?.results?.length) {
+                                    <jhi-programming-exercise-instructor-status
+                                        class="repository-status-icon me-2"
+                                        [participationType]="detail.data.type"
+                                        [participation]="detail.data.participation!"
+                                        [exercise]="detail.data.exercise"
+                                    ></jhi-programming-exercise-instructor-status>
+                                }
+                                @if (detail.data.participation?.results && detail.data.exercise.isAtLeastEditor) {
```

Comment:
I think you need to check for ...results.length. However, i would wrappe this if statement in the statement above (line 118).